CMakeLists.txt

```
@@ -16,7 +16,7 @@ endif()
 project(jwt-cpp)
 
 option(JWT_BUILD_EXAMPLES "Configure CMake to build examples (or not)" ON)
-option(JWT_BUILD_TESTS "Configure CMake to build tests (or not)" OFF)
+option(JWT_BUILD_TESTS "Configure CMake to build tests (or not)" ON)
```

The idea here is that consumer of the project can simply install the project which getting bother by the extra requirements of the tests which is why this is off by default

```suggestion
option(JWT_BUILD_TESTS "Configure CMake to build tests (or not)" OFF)
```

---

Yes, I have reverted this when testing was completed